### DISCUSSION TITLE
Is there a de-privacy option?

### DISCUSSION THREAD

**Discussion by KurtBliss** [FreeTubeApp/FreeTube#4738] [Q&A]
I want my watch history to sync up with my youtube account. I'm not sure if there's a setting i'm missing or maybe a fork? I doubt this is a thing considering the extra programming that would prbably be involved but Dang its the only thing that's been keeping me using the official youtube app and site for videos. The memory ussage on freetube is so low is crazy, and I also like the looks of the profile features.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
FreeTube would never implement an option to login with your YT account. This is a privacy first application and implementing this would defeat it being private.
